backend/src/Squidex.Domain.Apps.Entities/Contents/Queries/ContentQueryService.cs

```diff
@@ -91,10 +92,14 @@ public async Task<IResultList<IEnrichedContentEntity>> QueryAsync(Context contex
             }
 
             var schema = await GetSchemaOrThrowAsync(context, schemaIdOrName);
+            var permissionReadOwn = Permissions.ForApp(Permissions.AppContentReadOwn, context.App.Name, schemaIdOrName);
 
-            if (context.Permissions.Any(p => p.Id == Permissions.AppContentReadOwn))
+            if (context.Permissions.Any(p => {
```

Comment:
There is a method for that. You only need context.Permissions.Allows(permissionReadOwn)

but you also need to check if the user does not have the normal read permission.